docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -18,136 +18,98 @@ This article covers how you can get started with using F# on .NET Core 1.0 with
 
 ## Prerequisites
 
-To begin, you must install the [.NET Core SDK 1.0.0 - Preview 2 (build 003131)](https://dot.net/core).  There is no need to uninstall a previous version of the .NET Core SDK, as it supports side-by-side installations.
+To begin, you must install the [.NET Core SDK 1.0.3 - (build 004769 or later)](https://dot.net/core).  There is no need to uninstall a previous version of the .NET Core SDK, as it supports side-by-side installations.
 
 This article assumes that you know how to use a command line and have a preferred text editor.  If you don't already use it, [Visual Studio Code](https://code.visualstudio.com) is a great option as a text editor for F#.  To get awesome features like IntelliSense, better syntax highlighting, and more, you can download the [Ionide Extension](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp).
 
 ## Building a Simple Multi-project Solution
 
-1. Open a Command Line/Terminal.
-2. Create a new directory named `FSNetCore`.  Open Visual Studio code or your preferred editor inside this directory. 
-3. Under `FSNetCore`, create `src` and `test` directories.
-4. Under `FSNetCore`, create a new file called `global.json`.  It should have this as its contents:
+Open a command prompt/terminal and use the `dotnet new` command to create new solution file called `FSNetCore`:
 
-```json
-{
-    "projects":[ "src", "test" ]
-}
+```bash
+dotnet new sln -o FSNetCore`
 ```
 
-Your solutions structure should now look like this:
+Change directories to `FSNetCore` and start adding projects to the solution folder.
```

Comment:
The *test* directory is not needed or used in this walkthrough. The `dotnet new console -lang F# -o src/Library` creates the *src* directory. I will add "Here is what the directory structure looks like after running the command" to help illustrate.